### PR TITLE
Improve Error Message for Non-Linux OS Support in Keploy

### PR DESCRIPTION
**Pull request by SYSHIL**
## Related Issue
Given that its only a one-line change I don't think an issue is necessary but I can create one if its needed.
I'm totally new to the project and I encountered this while setting up keploy myself.

#### Describe the changes you've made
Updated the error message in [`core_service_others.go`]
This change helps users understand the requirement for running Dockerized versions of their applications when using Keploy on non-Linux operating systems

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Please let us know if any test cases are added

NA

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
NA

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| 
image
** | **
image
** |

### CODE REVIEW

**Comment by SYSHIL** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by SYSHIL** [comment]
recheck

**Comment by SYSHIL** [comment]
I'm not 100% sure if this is really needed but @Swpn0neel could u please take a look?

File: cli/provider/core_service_others.go [hunk 1]
```diff
@@ -33,9 +33,9 @@ func Get(ctx context.Context, cmd string, c *config.Config, logger *zap.Logger,
 
 	if (cmd == "test" && c.Test.BasePath != "") || cmd == "normalize" || cmd == "templatize" {
 		return replaySvc, nil
-	}	
+	}
 
-	return nil, errors.New("command not supported in non linux os")
+	return nil, errors.New("command not supported in non linux os. if you are on windows or mac, please use the dockerized version of your application")
```
Comment (Sarthak160) [inline cli/provider/core_service_others.go:38]: You can also say please use wsl or dockerized version of your app.

**Review by Sarthak160 — APPROVED** [review]
LGTM

**Comment by PranshuSrivastava** [comment]
Hey @SYSHIL, please fix the merge conflicts so that this PR can be merged.

**Pull request merged**